src/components/Dialogs/Conditions/OrgUnitCondition.js

```diff
@@ -72,6 +79,111 @@ const OrgUnitCondition = ({
         />
     )
 }
+const getMetaDataId = (id) =>
+    ouIdHelper.hasLevelPrefix(id) || ouIdHelper.hasGroupPrefix(id)
```

Comment:
You don't need to do these checks before calling `removePrefix` since that function does those checks internally.

Reply:
As in, it should work to just have:

`getMetaDataId = id => ouIdHelper.removePrefix(id)`